### ISSUE TITLE
archivematica-dashboard is a dependency to archivematica-mcp-client

### ISSUE THREAD

**Issue by mariushusnes** [archivematica/Issues#564]
**Expected behaviour**
Install archivematica-mcp-client
Run archivematica-mcp-client

**Current behaviour**
When installing archivematica-mcp-client as a standalone instance, two errors arise when trying to run it:
ImportError: No module named components
ImportError: No module named accounts

These modules are a part of the archivematica-dashboard and the issue is resolved when installing archivematica-dashboard as well.

**Steps to reproduce**
Install archivematica-mcp-client
Run archivematica-mcp-client
Take note of errors
Install archivematica-dashboard
Successfully run archivematica-mcp-client

**Your environment (version of Archivematica, OS version, etc)**
CentOS Linux release 7.6.1810 (Core) 
archivematica-mcp-client-1.9.0-1.x86_64

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Verified in Waffle:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

**Comment by sverrewl** [comment]
Still experimenting. We expect that multiple MCP clients is desirable in the future. Not a issue installing the extra package. Obviously the install doc could use a update, at least for now.

**Comment by mariushusnes** [comment]
As @sverrewl already clarified, we expect to run multiple MCP clients to improve performance in the future and now that we are aware of the dependency, we can apply the workaround. The install docs for distributing components on multiple machines could use some more depth to ease the process.

**Comment by mamedin** [comment]
@mariushusnes The mcp-client-base dockerfile could give you some useful information about dependencies when this service runs as standalone: 
https://github.com/artefactual/archivematica/blob/stable/1.9.x/src/MCPClient-base.Dockerfile

**Comment by mariushusnes** [comment]
@mamedin I had a look at that earlier, but neither that dockerfile nor the docs tell us that you need to have archivematica-dashboard installed to be able to run archivematica-mcp-client.

**Comment by mamedin** [comment]
@mariushusnes Yes, this dependency is not explicit in docs or as package dependency. In documentation this dependency is fixed following the installation order. The mcp-client standalone service is not contemplated in docs or package.

**Comment by jorikvankemenade** [comment]
There is a similar problem for the MCP service. The [Dockerfile](https://github.com/artefactual/archivematica/blob/48409e31b72d877b91efefd8c51dfd4c10c70b69/src/MCPServer.Dockerfile#L32) gives a clue, but some documentation would be helpful. 

I am currently experimenting with a distributed, multi-client setup, I think I could make a first draft for some documentation on how to do this. Would this be interesting for other people, and if so what would be a good location for that?

**Comment by sromkey** [comment]
HI @jorikvankemenade - a PR to the Archivematica docs would be very welcome! I'm thinking probably this page: https://github.com/artefactual/archivematica-docs/blob/1.10/admin-manual/installation-setup/customization/customization.rst and/or this: https://github.com/artefactual/archivematica-docs/blob/1.10/admin-manual/installation-setup/customization/scaling-archivematica.rst would be a good place.
